diff --git a/source/inui/core/window/appwin.d b/source/inui/core/window/appwin.d
index c01f539..11adc27 100644
--- a/source/inui/core/window/appwin.d
+++ b/source/inui/core/window/appwin.d
@@ -380,11 +380,13 @@ public:
     }
 
     string getWindowHandle() {
-        SDL_SysWMinfo info;
-        auto res = SDL_GetWindowWMInfo(window, &info);
-        if (info.subsystem == SDL_SYSWM_TYPE.SDL_SYSWM_X11) {
-            import std.conv : to;
-            return "x11:" ~ info.info.x11.window.to!string(16);
+        version(linux) {
+            SDL_SysWMinfo info;
+            auto res = SDL_GetWindowWMInfo(window, &info);
+            if (info.subsystem == SDL_SYSWM_TYPE.SDL_SYSWM_X11) {
+                import std.conv : to;
+                return "x11:" ~ info.info.x11.window.to!string(16);
+            }
         }
         return "";
     }
